lib/fileutils.rb

```diff
@@ -1642,7 +1642,13 @@ def install(src, dest, mode: nil, owner: nil, group: nil, preserve: nil,
       st = File.stat(s)
       unless File.exist?(d) and compare_file(s, d)
         remove_file d, true
-        copy_file s, d
+        if d.to_s.end_with?('/')
```

Comment:
`.to_s` using with strings here is redundant

Reply:
Thanks! Confirmed.